gen/pb-rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sigstore_protobuf_specs"
-version = "0.1.0-rc.1"
+version = "0.1.0-rc.2"
```

Comment:
@woodruffw @jleightcap I assumed we'd just bump the RC until a stable 0.1.0 is released, lemme know if you want to do something else